### PR TITLE
Add narrator menu hotkey text

### PR DESCRIPTION
**Pull request by boholder**

### CODE REVIEW

**Review by khanshoaib3 — APPROVED** [review]

File: en_us.json [hunk 1]
```diff
@@ -171,18 +172,18 @@
   "minecraft_access.gui.screen.value_entry_menu": "Value Entry Menu",
   "minecraft_access.gui.value_entry_menu.info_text": "Press enter key to accept and escape key to cancel.\tValue: %s",
   "minecraft_access.gui.screen.f4_menu": "F4 Menu",
-  "minecraft_access.narrator_menu.gui.button.block_and_fluid_target_info": "Block and Fluid target information",
-  "minecraft_access.narrator_menu.gui.button.block_and_fluid_target_position": "Block and Fluid target position",
+  "minecraft_access.narrator_menu.gui.button.block_and_fluid_target_info": "1. Block and Fluid target information",
+  "minecraft_access.narrator_menu.gui.button.block_and_fluid_target_position": "2. Block and Fluid target position",
   "minecraft_access.narrator_menu.gui.button.entity_target_info": "Entity target information",
   "minecraft_access.narrator_menu.gui.button.entity_target_position": "Entity target position",
```
Comment (boholder) [inline en_us.json:178]: Here are two config what I mentioned as "unused", also can not find a hit when running a full text search inside main project code. @khanshoaib3

Comment (khanshoaib3) [reply en_us.json:178]: They are being used, here: https://github.com/khanshoaib3/minecraft-access/blob/ee82d1ca0f9b36067e07bcf5d49c0ba9b15c6927/common/src/main/java/com/github/khanshoaib3/minecraft_access/features/narrator_menu/NarratorMenuGUI.java#L42

Comment (boholder) [reply en_us.json:178]: Oh, sorry, I mean entity info and entity position...

Comment (khanshoaib3) [reply en_us.json:178]: Oh those, I implemented these earlier but they weren't working so I removed them, I guess I forgot removing the i18n entries. Will remove them, thanks.

**Pull request merged**